arquidiocesis-app/src/screens/alertas/Alertas.js

```diff
@@ -0,0 +1,76 @@
+/* 
+Nombre: Grupos.js
+Usuario con acceso: Admin, acompañante, coordinador
+Descripción: Pantalla para ver los grupos HEMA
+*/
+import React from 'react';
+import { View, Text, ScrollView } from 'react-native';
+import { List } from '../../components';
+
+export default () => {
+  // var [user, setUser] = useState(null);
+  // var [data, setData] = useState(false);
+  // var [refreshing, setRefreshing] = useState(false);
+  // var [error, setError] = useState(false);
+  const alert = {
```

Comment:
Yo sugeriría declarar de una vez un arreglo estático, sin usar el constructor de Objetos, esto sólo para mejorar la legibilidad y en caso de que queramos llenar la lista para la prueba/muestra con los profes. Podemos hacer un copy paste, o extraer dinamicamente la "mini fabrica" de alertas, sin repetir código. Es un pequeño patrón 😄 
```suggestion
const alerts = [
      {
        name: 'alerta 1',
        description: 'descripción corta',
        date: '18/10/21'
      },
      {
        name: 'alerta 2',
        description: 'descripción corta',
        date: '17/10/21'
      }
    ];
```